plugins/out_cloudwatch_logs/cloudwatch_logs.c

```diff
@@ -524,6 +570,18 @@ static struct flb_config_map config_map[] = {
      "Specify a custom endpoint for the STS API, can be used with the role_arn parameter"
     },
 
+    {
+     FLB_CONFIG_MAP_STR, "metric_namespace", NULL,
+     0, FLB_FALSE, 0,
+     "Metric namespace for CloudWatch EMF logs"
+    },
+
+    {
+     FLB_CONFIG_MAP_STR, "metric_dimensions", NULL,
+     0, FLB_FALSE, 0,
+     "CloudWatch metric dimension sets as a comma and semicolon seperated string"
```

Comment:
You comment inside of the `flb_cloudwatch` structure was better. Please copy that or write a longer similar explanation here that will allow the user to understand the lists of lists idea.

Reply:
Updated.